### DISCUSSION TITLE
Someones got into my phone and emails and taken email's

### DISCUSSION THREAD

**Discussion by jcutty87-hacked** [community/community#38120] [Other Feature Feedback, Questions, & Ideas] [answered]
Someone's been messing with my phone.  Ive been hacked and shes is doing this.. I just found  this hut thing  under my microsoft log in ummn how can I learn how to protect myself?; from her gaining  any more control.. like this girl plays dumb but i knoe her friends are helping her.   What can she gain outta stealing emails and manipulation of my phone.   I have  517 apps on my phone!?!?! Can someone help me or guide me through this hackin shit  please 🙏

**Comment by queenofcorgis** [comment, marked as answer]
Hi @jcutty87-hacked 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
